meson_options.txt

```diff
@@ -40,6 +40,12 @@ option(
     value : 'auto',
     description : 'Whether to build support for manipulating the utmp/utmpx database via the related POSIX functions.'
 )
+option(
```

Comment:
Thanks for adding new option to meson. Please also add it to `MESON_BUILD` doc.

Reply:
This is the only thing outstanding before merging I think.

Reply:
alright, min

Reply:
done

Reply:
Merged! Thanks